### PR TITLE
Fx53 changes

### PR DESCRIPTION
**Pull request by chrisdavidmills**
This PR includes a general copy edit of all the existing feature entries, plus I've added in an entry for font-variation-settings.

### CODE REVIEW

**Comment by marco-c** [comment]
LGTM. @teoli2003 do you want to review too?

File: features/css-display-flow-root.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: CSS display: flow-root
```
Comment (marco-c) [inline features/css-display-flow-root.md]: The colon character is a special character in YAML. Can you quote the string?

`title: "CSS display: flow-root"`

**Comment by chrisdavidmills** [comment]
Sure, fixed it

**Comment by marco-c** [comment]
Thanks!

**Pull request merged**